wordle-web/pages/game.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <v-row>
+        <v-col>
+            <v-card>
+
+            <h2>Game Same</h2>
+            <p>
+                this is a game that we play with words and ideas to find the right answer.
+               
+            </p>
+             <v-btn  color="red" nuxt to ="/"> GO BACK
```

Comment:
```suggestion
             <v-btn  color="error" nuxt to ="/"> GO BACK
```

Consider using your preset theme colors. I know we didn't talk about this until after Assignment 1 though, so good job with setting the colors!